### ISSUE TITLE
AI in Java!

### ISSUE THREAD

**Issue by Nazchanel** [joshdimatteo/ai-from-scratch#1]
Never expected to see a neural network in Java! If you are actually serious about AI try python. There you have access to Tensorflow. 

Check out my AI project:
[github.com/nazchanel/Fake_News_Detection](url)

**Comment by joshdimatteo** [comment]
@Nazchanel yea its an old project but its pretty cool lmao I enjoy coding stuff from scratch instead of using prebuilt libraries but if I find a good way to learn it I'll probably give tensorflow a shot.

**Comment by Nazchanel** [comment]
It isn't as hard as you think to learn Tensorflow. Just learn the terminology (epochs, learning rate, etc) and what they mean. That will make it much easier for you and flatten the learning curve. Also, [sci-kit learn](https://scikit-learn.org) is a great library to use if you are using *Python*. 

I would recommend *Python* for any machine learning because, first, there are so many libraries, and, second, the processing power. 

Regarding *Tensorflow*, you can use either *C++* or *Python*. 

Also to learn *Tensorflow*, watch this [video](https://www.youtube.com/watch?v=tPYj3fFJGjk) for a deep understanding of it. 

You don't need to watch the whole ***6 hour*** video but just skim through to the parts that you need.

*Hope this helped*
- Eshan I.
